generate.sh

```diff
@@ -45,6 +48,13 @@ cd "$project_dir" || exit
 
 # Remove files that are specific to the scaffold but not to projects
 rm CONTRIBUTING.md ROADMAP.md web/DEVELOPING.md web/CHANGELOG.md
+sed -i -e "s/Merlin's Potions/$project_name/g" web/app/containers/header/partials/header-title.jsx
```

Comment:
Can we add a blank line here. These lines aren't related to the `#Remove files that are ...` line and comment.